chia/consensus/blockchain.py

```diff
@@ -546,6 +546,8 @@ async def _reconsider_peak(
 
         if genesis:
             records_to_add = [block_record]
+        elif fork_info.block_hashes == [block_record.header_hash]:
+            records_to_add = [block_record]
```

Comment:
Perhaps `if genesis or fork_info.block_hashes == [block_record.header_hash]:` ?

If it's getting its own branch to bring attention to it, I would add an explanatory comment.

Reply:
+1 on explaining why we need this new case